Mesh_3/include/CGAL/Mesh_3/features_detection/features_detection.h

```diff
@@ -412,7 +412,7 @@ std::vector<std::vector<P>> poly00001122(const int prec = 10)
 template<typename P>
 std::vector<std::vector<P>> poly00010121(const int prec = 10)
 {
-    auto y = [](double z) { return ((3 * z * z - 1) - std::sqrt(CGAL::square(1 - 3 * z * z) - 12 * (z - 1) * z * z)) / (6 * (z - 1) * z); };
+    auto y = [](double z) { return ((3 * z * z - 1) - std::sqrt((1. - 3 * z * z)*(1. - 3 * z * z) - 12 * (z - 1) * z * z)) / (6 * (z - 1) * z); };
```

Comment:
I disagree to that change, to avoid a few operations. Add:
```c++
#include <CGAL/number_utils.h> // for CGAL::square
#include <CGAL/double.h>       // for Algebraic_structure_traits<double>
```
in the list of included headers, and `CGAL::square` for `double` will then compile correctly. Given the error message [in the CI log](https://github.com/CGAL/cgal/actions/runs/3203394248/jobs/5233436469#step:4:5211), I think `<CGAL/number_utils.h>` is already included, but not `<CGAL/double.h>`.

Reply:
what's the point? this will not change anything in the end.